include/tc/lang/lexer.h

```diff
@@ -418,9 +418,9 @@ struct Lexer {
 
  private:
   Token lex() {
-    int kind;
-    size_t start;
-    size_t length;
+    int kind = -1;
```

Comment:
I initialized these values by looking at other contructors for SourceRange in the code and also ran the C++/python test afterwards. @zdevito , is this safe to do?